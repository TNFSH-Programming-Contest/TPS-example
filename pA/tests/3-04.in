180430342 / 730333328
