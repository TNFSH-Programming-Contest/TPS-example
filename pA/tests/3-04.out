180430342 / 730333328 = 0 ... 180430342
